### PR TITLE
Fix padding in itemdetails' selectors

### PR DESCRIPTION
**Pull request by ferferga**
Fixes the overlapping and the small separation between the selectors in the video elements (version, subtitles, audio, etc...)

**Before:**
Sin título_old

**After:**
Sin título

### CODE REVIEW

**Comment by ferferga** [comment]
Switched to classes and margins to get the exact same effect. 

I thought that only adding the padding to the video would fix all the spacing issues, but in mobile layout there was still spacing issues between the audio and subtitles selection tracks. Using padding would misplace the dropdown arrow, so I switched to margins as well.

With margins there are no issues in TV, Desktop and Mobile

File: src/itemdetails.html [hunk 1]
```diff
@@ -116,14 +116,14 @@
                         <div class="genres hide" style="margin: .7em 0;font-size:92%;"></div>
                         <div class="directors hide" style="margin: .7em 0;font-size:92%;"></div>
 
-                        <form class="trackSelections flex align-items-center flex-wrap-wrap hide focuscontainer-x" style="margin: .5em 0;font-size:92%; padding: 0;">
-                            <div class="selectContainer selectContainer-inline selectSourceContainer hide trackSelectionFieldContainer flex-shrink-zero" style="margin-right:1.5em;">
+                        <form class="trackSelections flex align-items-center flex-wrap-wrap hide focuscontainer-x" style="padding:1em 0">
```
Comment (dkanada) [inline src/itemdetails.html]: Why does only this one have padding?

Comment (ferferga) [reply src/itemdetails.html]: Good catch! @dkanada

**Comment by ferferga** [comment]
Saw that the text was noticeable misaligned from the other texts in TV layout (a little bit in the others as well) , so removed left margin and now LGTM.

**Pull request merged**